spikes/aks-secure-baseline/charts/ngsa/templates/deployment.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "ngsa.fullname" . }}
+  labels:
+    app.kubernetes.io/component: app
+    {{- include "ngsa.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
```

Comment:
This file was copied from ngsa helm chart and modified. Only change here was to remove references to keda variables.